arduino-core/src/cc/arduino/contributions/ConsoleProgressListener.java

```diff
@@ -33,12 +33,19 @@
 
 public class ConsoleProgressListener implements ProgressListener {
   private String lastStatus = "";
+  private double lastProgress = 0.0;
 
   @Override
   public void onProgress(Progress progress) {
-    if (!lastStatus.equals(progress.getStatus())) {
+    // Reduce verbosity when running in console
+    String s = progress.getStatus().replaceAll("[0-9]", "");
+    double p = progress.getProgress();
```

Comment:
The point of this change is to reduce the output of the lines like:
```
Downloading tools 10k of 15680k...
Downloading tools 20k of 15680k...
Downloading tools 30k of 15680k...
Downloading tools 40k of 15680k...
```

While this is nice in a GUI, with a progress bar etc., it's a mess on command line, this commit was the simplest way that comes to mind.